frigate/util.py

```diff
@@ -383,6 +384,13 @@ def clipped(obj, frame_shape):
         return False
 
 
+def restart_frigate():
+    for proc in psutil.process_iter():
```

Comment:
Rather than setting off a chain reaction by terminating the detection processes, you should be able to just call `os.kill(os.getpid(), signal.SIGTERM)`.

Reply:
ok, it's changed